src/windows/wslaservice/exe/WSLAVirtualMachine.h

```diff
@@ -100,6 +105,12 @@ class DECLSPEC_UUID("0CFC5DC1-B6A7-45FC-8034-3FA9ED73CE30") WSLAVirtualMachine
     std::wstring m_debugShellPipe;
 
     wsl::windows::common::hcs::unique_hcs_system m_computeSystem;
+
+    std::filesystem::path m_vmSavedStateFile;
```

Comment:
WSL itself has an entire temp directory that is cleaned up when the VM exits. That might make this a bit cleaner and be useful in future cases where we need files to stick around for the lifetime of the VM.